app/views/pages/TaxCodeView.scala.html

```diff
@@ -57,7 +57,7 @@
 
         @govukDetails(Details(
             summary = Text(messages("label.tax-code")),
-            content = HtmlContent(Messages("quick_calc.salary.question.tax_code_info_url_a") + link("https://www.gov.uk/tax-codes/overview", "quick_calc.salary.question.tax_code_info_url_b"))
+            content = HtmlContent(Messages("quick_calc.salary.question.tax_code_info_url_a") + " <a href=\"https://www.gov.uk/tax-codes/overview\" class=\"govuk-link\">" + Messages("quick_calc.salary.question.tax_code_info_url_b") + "</a>.")
```

Comment:
Not sure this will open in new tab (like the injected component, any reason we are not using the injected component?